File: src/test/java/dgroomes/AppTest.java
package dgroomes;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class AppTest {
    @Test
    void appHasAGreeting() {
        var message = new MyMessage("Hello world!");

        assertEquals("Hello world!", message.message());
    }
}

File: pre-compiled-script-plugin/app/src/test/kotlin/dgroomes/pre_compiled_script_plugin/AppTest.kt
package dgroomes.pre_compiled_script_plugin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class AppTest {

    @Test
    fun describeGreeting() {
        assertEquals(
            """{"message":"hello"}""",
            jsonifyGreeting(Greeting("hello")),
        )
    }
}
